src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -1210,7 +1236,14 @@ struct MHAHelper {
                                        ov::element::f32,
                                        ov::element::f32,
                                        alibi_slope);
-        });
+        };
+        if (prefer_static_loop) {
+            parallel_for3d(B, kv_len_in_blocks, loop_hk ? _Hk : _H, loop_qk);
```

Comment:
The last dim of `parallel_for` is the same for several `parallel_for`, please consider to set them in a common place.
```
size_t h_dims = loop_hk ? _Hk : _H, loop_qk
```

Reply:
Done.